README.md

```diff
@@ -56,6 +56,16 @@ get tricky, so please leave this directory out of your commits and pull request
 
 The committer doing the final merge will generate the `content/` directory at that time.
 
+### Running using Docker
+
+**Note:** This is an optional method if you don't want to install gems locally and just use a container for development.
+
+Make sure you have docker installed and we'll use the official jekyll image to build the site.
```

Comment:
Please make sure you have Docker [installed](link) on your local machine.

docker -> Docker
jekyll -> Jekyll

Move the image piece in the `serve` paragraph.